collector/mdadm_linux.go

```diff
@@ -158,19 +175,28 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 		}
 		currentMD = mainLine[0]               // The name of the md-device.
 		isActive := (mainLine[2] == "active") // The activity status of the md-device.
-		personality = mainLine[3]             // The personality type of the md-device.
+		personality = ""
+		for _, possiblePersonality := range mainLine {
+			if raidPersonalityRE.MatchString(possiblePersonality) {
+				personality = possiblePersonality
+				// break
```

Comment:
Are multiple personalities supported? are we just aiming to match the last?
Either uncomment the break, or remove it